MAUI/TextInputLayout/Getting-Started.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control, its elements and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Getting Started with .NET MAUI Text Input Layout (SfTextInputLayout)
+
+This section explains the steps required to configure the `.NET MAUI Text Input Layout` (SfTextInputLayout) control with floating label.
+
+## Creating an application using the .NET MAUI Text Input Layout
+
+This section explains the steps required to work with the `SfTextInputLayout` control for .NET MAUI.
+
+## Adding .NET MAUI Text Input Layout reference
+
+The Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add `.NET MAUI Text Input Layout` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and then install it.
+
+## Handler registration 
+
+ In the MauiProgram.cs file, register the handler for Syncfusion core.
+
+{% highlight c# hl_lines="6 17" %}   
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace TextInputLayoutSample
+{
+  public static class MauiProgram
+  {
+	public static MauiApp CreateMauiApp()
+	{
+		var builder = MauiApp.CreateBuilder();
+		builder
+		.UseMauiApp<App>()
+		.ConfigureSyncfusionCore()
+		.ConfigureFonts(fonts =>
+		{
+			fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+		});
+
+		return builder.Build();
+	 }
+   
+  }
+
+}     
+
+{% endhighlight %}
+
+
+## Adding a namespace
+
+Add the following namespace to add `.NET MAUI Text Input Layout.
```

Comment:
Add the following namespace to add `.NET MAUI Text Input Layout.'

Reply:
Addressed the changes